### PR TITLE
mkdir: add Farsi translation

### PR DESCRIPTION
**Pull request by MohammadRostamiorg**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by spageektti** [comment]
Hi. We don't have a page for `mkdir` for `linux`, only `common` and `windows`.

File: pages.fa/common/mkdir.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# mkdir
+
+> Create directories and set their permissions.
+> More information: <https://www.gnu.org/software/coreutils/mkdir>.
```
Comment (spageektti) [inline pages.fa/common/mkdir.md:4]: You need to translate these lines too.

Comment (MohammadRostamiorg) [reply pages.fa/common/mkdir.md]: translated

**Review by spageektti — APPROVED** [review]
LGTM, thanks for your contribution.

File: pages.fa/common/mkdir.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# mkdir
+
+> ساخت پوشه ها و مجوز تنظیم مجوز آنها.
+> اطلاعات بیشتر: <https://www.gnu.org/software/coreutils/mkdir>.
+
+- ساخت پوشه خاص:
+
+`mkdir {{path/to/directory1 path/to/directory2 ...}}`
```
[the remark this answers is not in the document]
Comment (MrMw3) [reply pages.fa/common/mkdir.md:8]: The first occurrence of the word "مجوز" in line 3 seems to be extra, please fix it and commit again. Thank you.

**Comment by MrMw3** [comment]
The first occurrence of the word "مجوز" in line 3 seems to be extra, please fix it and commit again. Thank you.

File: pages.fa/common/mkdir.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (MohammadRostamiorg) [reply pages.fa/common/mkdir.md:8]: persian (Farsi) is rtl language and it's doesn't looks good when i wanna to translate the placeholders

**Comment by MohammadRostamiorg** [comment]
> The first occurrence of the word "مجوز" in line 3 seems to be extra, please fix it and commit again. Thank you.

done , I'm sorry about mistake

File: pages.fa/common/mkdir.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (spageektti) [reply pages.fa/common/mkdir.md:8]: None of the placeholders are translated under pages.fa directory.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by MrMw3 — APPROVED** [review]
LGTM.

**Pull request merged**